src/commands/Twitch/twitchsubscription.ts

```diff
@@ -1,16 +1,18 @@
 import { chunk } from '@klasa/utils';
```

Comment:
Twitch needs extra thorough testing. I don't want it to be broken again after this PR.